### PR TITLE
ENT-6490: Update Node Docker instruction for OS 4.9

### PR DESCRIPTION
**Pull request by pmck91**
An update to @vkolomeyko pull request [624](https://github.com/corda/corda-docs-portal/pull/624).

### CODE REVIEW

**Comment by vkolomeyko** [comment]
Looks fine to me, however CI build fails at the moment:
```
[2022-05-24T19:00:27.342Z] Error: Error building site: "/src/content/en/platform/corda/4.9/community/node-docker-deployments.md:6:1": failed to unmarshal YAML: yaml: line 5: mapping values are not allowed in this context
```
This is likely to be caused by spaces removed in the header of the file.

**Review by vkolomeyko — CHANGES_REQUESTED** [review]
.

**Review by vkolomeyko — DISMISSED** [review]
LGTM

File: content/en/platform/corda/4.9/community/node-docker-deployments.md [hunk 1]
```diff
@@ -138,56 +138,35 @@ There are some optional variables which allow customisation of the generated con
 * `MY_P2P_PORT` - The port to advertise the node on (defaults to 10200). If changed, ensure the container is launched with the correct published ports.
 * `MY_RPC_PORT` - The port to open for RPC connections to the node (defaults to 10201). If changed, ensure the container is launched with the correct published ports.
 
-Once the container has finished performing the initial registration, the node can be started as normal
+Once the container has finished performing the initial registration, it is necessary to initialize node's DB as follows:
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.9/community/node-docker-deployments.md:141]: you must initialize the node's DB as follows:

File: content/en/platform/corda/4.9/community/node-docker-deployments.md [hunk 2]
```diff
@@ -138,56 +138,35 @@ There are some optional variables which allow customisation of the generated con
...
 
-# Performing Database Migrations
-
-The image contains the database-migration tool. It is possible to run this in two modes within a container.
-
-## Generating Migration Jars
-
-In this mode, the database-migration tool will scan the provided CorDapps, and generate corresponding migration jars. These jars will be placed alongside
-the source CorDapps. In this example, there are two CorDapps provided `corda-insurance.jar` and `corda-kyc.jar`
+After that the node can be started as normal:
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.9/community/node-docker-deployments.md:158]: You can now start the node as normal:

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

**Review by nadinequinr3 — APPROVED** [review]

**Review by vkolomeyko — APPROVED** [review]

**Comment by vkolomeyko** [comment]
@nadinequinr3 @benpesterr3 - would you be able to merge this PR? (as I do not have permissions)

**Pull request merged**